MODULE.bazel

```diff
@@ -0,0 +1,37 @@
+module(
+    name = "wasmedge_go",
+    version = "0.14.0",
+)
+
+bazel_dep(name = "platforms", version = "0.0.11")
+bazel_dep(name = "rules_go", version = "0.50.1", repo_name = "io_bazel_rules_go")
+bazel_dep(name = "gazelle", version = "0.38.0", repo_name = "bazel_gazelle")
+
+go_sdk = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.download(version = "1.23.0")
+use_repo(go_sdk, "go_toolchains")
+
+register_toolchains("@go_toolchains//:all")
+
+go_deps = use_extension("@bazel_gazelle//:extensions.bzl", "go_deps")
+go_deps.from_file(go_mod = "//:go.mod")
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "wasmedge_c",
+    build_file_content = """
+cc_library(
+    name = "wasmedge_c",
+    srcs = select({
+        "@platforms//os:linux": ["lib/libwasmedge.so"],
+        "@platforms//os:windows": ["lib/wasmedge.lib"],
+        "//conditions:default": ["lib/libwasmedge.so"],
```

Comment:
I found that you are adding Windows here. Why do we only have "linux" in the URLs? Shouldn't it be "linux," "macos," and "windows"?